.dev/CRAN_Release.cmd

```diff
@@ -195,15 +195,15 @@ R CMD build .
 export GITHUB_PAT="f1c.. github personal access token ..7ad"
 # avoids many too-many-requests in --as-cran's ping-all-URLs step (20 mins) inside the `checking CRAN incoming feasibility...` step.
 # Many thanks to Dirk for the tipoff that setting this env variable solves the problem, #4832.
-R CMD check data.table_1.14.99.tar.gz --as-cran
```

Comment:
this commit is somewhat strange to survive to `master`, but the changes here still look useful. @TysonStanley shall we add this commit (perhaps with a better commit message) directly to `master` instead of merging along with `1-15-99`?

Reply:
#5938